Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelCancelEditingTests.swift

```diff
@@ -0,0 +1,36 @@
+import XCTest
+
+final class SitterProfileViewModelCancelEditingTests: XCTestCase {
+    func testCancelEditingMethodExistsInAPI() {
```

Comment:
```suggestion
    func testExistsInAPI() {
```
nit: You have not to write the name of the method in functions, if the name of the test class unambiguously indicates a method under tests.

Reply:
I followed the general naming style that is used in the project. But, I'll keep that in mind for the future, thanks.